src/test/resources/solr/dssolr/conf/schema.xml

```diff
@@ -362,8 +362,13 @@ In solrconfig.xml the classic schema is defined with this:  <schemaFactory class
   </field>
 
   <field name="resource_id" type="string" multiValued="true" stored="true">
-    <?description URI pointing to the digital representation of the resource?>
-    <?example     https://example.com/imageserver/exampleCollection/examplePart/exampleCatalog/0000/123/456/AA123456?>
+    <?description ID used to construct URI for resource.?>
+    <?example     exampleCollection/examplePart/exampleCatalog/0000/123/456/AA123456?>
+  </field>
+
+  <field name="url" type="string" stored="true">
```

Comment:
`url` is quite non-descriptive. It could just as well point to a page describing the material as it could point to a bitmap. In this case it is a thumbnail (or a preview), so maybe the field name could reflect that?